modules/contentbox/models/content/BaseContent.cfc

```diff
@@ -1236,7 +1236,7 @@ component 	persistent="true"
 	* Build content cache keys according to sent content object
 	*/
 	string function buildContentCacheKey(){
-		return "cb-content-#cgi.http_host#-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#";
+		return "cb-content-#cgi.http_host#-#cgi.query_string#-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#";
```

Comment:
Ok, I see your value now.  I would suggest the following then. create an input hash to make this easier.

```
var inputHash = hash( cgi.http_host & cgi.query_string );
return "cb-content-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#-#inputHash#";
```
I would also add it to the end. This way we can still run pattern abased clearing of the cache if needed.